### ISSUE TITLE
bug: Using Jan settings to clearing Logs Stops New Log Generation until App Restart

### ISSUE THREAD

**Issue by Van-QA** [janhq/jan#2730]
**Describe the bug**
When users clear their logs using the settings in Jan, the application stops generating any new logs until the user restarts the Jan app.

**Steps to reproduce**
1. Open Jan application.
2. Go to the settings menu.
3. Click on the 'Clear Logs' option.
4. Confirm the action in the pop-up window.
5. Notice that no logs remain, as expected.
6. Continue to use Jan but no new logs will be created, even for errors or important events.
7. Restart the application.
8. Check that new logs are being generated and recorded.

**Expected behavior**
After users clear their logs using the settings menu, Jan should continue to generate new logs for any errors or important events that occur.

**Screenshots**
![image](https://github.com/janhq/jan/assets/64197333/74474e05-6b16-44ea-80c9-a9d319e3585c)

**Logs**
[Since the expected behavior is for logs to continue being generated, there are no relevant logs to provide in this case.]

**Additional context**
The issue of not generating new logs after clearing the existing ones may cause inconvenience for users who want to track their progress or troubleshoot issues. It is recommended to investigate and fix this bug in future Jan updates to ensure continuous logging functionality.

**Comment by Propheticus** [comment]
For me, no server logs are written to server.log any more. (regardless any reset or restart of app).
Some of the server logs when starting are written to app.log but they are very limited. By no means 'verbose'.
```
2024-04-15T19:37:51.180Z [SERVER]::Debug: Starting JAN API server...
2024-04-15T19:37:54.134Z [APP]::Server listening at http://127.0.0.1:1337
2024-04-15T19:37:54.134Z [SERVER]::Debug: JAN API listening at: http://127.0.0.1:1337
```

**Comment by Van-QA** [comment]
Resolved as of Jan v0.4.11-386

**Comment by Van-QA** [comment]
Happening again on Jan v0.4.11-391 :x:
